libs/blocks/editorial-card/editorial-card.css

```diff
@@ -0,0 +1,251 @@
+
+.editorial-card,
+.editorial-card::after,
+.editorial-card::before {
+    box-sizing: border-box;
+}
+
+.editorial-card {
+  border: 1px solid var(--color-gray-300);
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  /* stylelint-disable-next-line property-no-vendor-prefix */
+  -webkit-transform: translateZ(0);
+}
+
+.section.masonry-layout > .editorial-card[class*='grid-span-'] {
+  display: flex;
+}
+
+.dark .editorial-card,
+.editorial-card.dark {
+  color: #fff;
+  border-color: var(--color-gray-700);
+}
+
+.editorial-card.no-bg { background: unset;}
+.editorial-card.no-border { border: unset;}
+
+.editorial-card.click {
```

Comment:
I agree, the `brick` block uses `.click` for the same variant feature so that's why I left it at that. Im open to change if need be or have both (one an alias)

Reply:
Better to keep things consistent in this case 👍